atom/renderer/lib/override.coffee

```diff
@@ -5,6 +5,11 @@ remote = require 'remote'
 # Window object returned by "window.open".
 class FakeWindow
   constructor: (@guestId) ->
+    that = this
```

Comment:
In CoffeeScript we can use fat arrow `=>` to get rid of `that`:
http://coffeescript.org/#fat-arrow